catt/controllers.py

```diff
@@ -69,20 +69,27 @@ def get_chromecast_with_ip(device_ip, port=DEFAULT_PORT):
         return None
 
 
-def get_cast(device_name=None):
+def get_cast(device=None):
```

Comment:
This could use a short docstring as well.

Reply:
Will do.